src/common/ui-components/WrappedMenu/WrappedMenu.tsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
```

Comment:
Looks like this file should be in domain and not common as it touches the apps state.

Reply:
A better name would be AhtiMenu?

Reply:
yes, I agree. How abour I do it in the next PR?